centreon/tests/e2e/cypress/e2e/Platform-update/01-platform-update.feature

```diff
@@ -0,0 +1,26 @@
+@execTimeout(200000)
+Feature: Update platform from version_A to version_B of the same MAJOR
+An admin user can update a platform, from a version A to version B
+which is higher than version A, within the same MAJOR.
+
+Scenario: Administrator performs a platform update procedure
+Given a running platform in '<version_A>'
+When administrator updates packages to '<version_B>'
+And administrator runs the update procedure
+Then monitoring should be up and running after update procedure is complete to '<version_B>' 
+
+Examples:
+  | version_A | version_B |
```

Comment:
I'm not sure it is the best usecase to have a version B
IMHO, version B should be the current code, cause the current tests only validate versions which are on stable repo
If there is an issue on upgrade with the current code, it is too late :)

Reply:
@kduret might need a little help in this